styles/main.css

```diff
@@ -59,37 +59,42 @@ main #useful-links a{
 footer {
     text-align: center;
     font-size: 12px;
-    margin: 0 0 0 3em;
+    margin: 0 0 3em 0;
 }
 
 .width-wrap {
     background-color: #7EAEE4;
-    border: 1px groove rgb(0,0,0,0.3);
-    width:1024px;
+    border: 1px solid rgba(0,0,0,0.3);
+    border-top: none;

```

Comment:
Lab asked for no top border
When specifying an alpha value in a a colour, you MUST use `rgba` and NOT `rgb`